views/widget.html

```diff
@@ -6,12 +6,25 @@
 <label for="{{ id }}" class="apos-forms-label {{ classPrefix + '__label' if classPrefix }}">
   {{ widget.fieldLabel}}
   {% if not widget.required %} {{ __('(Optional)') }}{% endif %}
-  <span class="apos-forms-label-message {{ classPrefix + '__label-message' if classPrefix }}" data-apos-input-message="{{ widget.fieldName}}" hidden></span>
+  <span class="apos-forms-label-message {{ classPrefix + '__label-message' if classPrefix }}" data-apos-input-message="{{ widget.fieldName }}" hidden></span>
 </label>
 
 <input class="apos-forms-input {{ classPrefix + '__input' if classPrefix }}"
   type="string"
-  id="{{ id }}" name="{{ widget.fieldName}}"
-  placeholder="{{ widget.placeholder}}"
-  {% if widget.required %}required{% endif %}
+  id="{{ id }}" name="{{ widget.fieldName }}"
+  placeholder="{{ widget.placeholder }}"
+  data-apos-forms-google-api-key="{{ data.googleApiKey }}"
+  {% if widget.required -%} required {%- endif -%}
 />
+
+{% if widget.splitAddress %}
+  {% for field in widget.splitAddress %}
+    </br>
+    <input class="apos-forms-input {{ classPrefix + '__input' if classPrefix }} apos-forms-input-{{ field }}"
+      type="string"
+      disabled="true"
```

Comment:
Broken out fields are disabled. I thought that was the thing to do, but open to suggestions.

Reply:
Not disabled anymore.